backend/app/services/user_service.py

```diff
@@ -46,8 +46,14 @@ async def pwd_reset(*, request: Request, obj: ResetPassword) -> int:
             np1 = obj.new_password
             np2 = obj.confirm_password
             if np1 != np2:
-                raise errors.ForbiddenError(msg='新密码输入不一致')
+                raise errors.ForbiddenError(msg='两次密码输入不一致')
```

Comment:
It seems that it can be verified in the schema

Reply:
It doesn‘t seem to meet the norms.

Schema is mainly used for data format verification, not business.